src/improvements/single.just

```diff
@@ -1,39 +1,40 @@
 set dotenv-load
 
-export rpcUrl := env_var('ETH_RPC_URL')
+# Common variables used across multiple recipes
+# Export variables in uppercase for shell script usage
+export TASK_PATH := invocation_directory()
+export SCRIPT_PATH := justfile_directory()
+
+# Get RPC URL based on task path
+rpc_cmd := SCRIPT_PATH + "/script/get-rpc-url.sh \"" + TASK_PATH + "\""
+export rpcUrl := shell(rpc_cmd)
+
+# Fetch the template name from the config.toml file
+cmd := "sed -n 's/^[[:space:]]*templateName[[:space:]]*=[[:space:]]*\\\"\\([^\\\"]*\\)\\\".*/\\1/p' " + TASK_PATH + "/config.toml"
+export SCRIPT_NAME := shell(cmd)
```

Comment:
We already have [yq](https://github.com/mikefarah/yq) as a dependency, which we can use to parse the config file in a way that's more readable then sed